Source/SocketIOClient.swift

```diff
@@ -275,6 +275,19 @@ public final class SocketIOClient : NSObject, SocketEngineClient, SocketParsable
         }
     }
 
+    ///ping & pong
+    public func engineDidSentPing() {
+        DefaultSocketLogger.Logger.error("%@", type: logType, args: "")
+        
+        handleEvent("sentPing", data: [""], isInternalMessage: true)
+    }
+    
+    public func engineDidReceivedPong() {
+        DefaultSocketLogger.Logger.error("%@", type: logType, args: "")
```

Comment:
What other info you require in logs? I will accommodate that. Furthermore closing this PR and opening a new one against development branch with the suggested changes.